src/main/java/net/datafaker/providers/movie/Avatar.java

```diff
@@ -14,6 +16,23 @@ protected Avatar(MovieProviders faker) {
     }
 
     public String image() {
-        return baseUrl + faker.regexify("[a-z]{8}") + ".png";
+        return image(DEFAULT_SET);
+    }
+
+    public String image(Set set) {
+        return baseUrl + faker.regexify("[a-z]{8}") + "?set=set" + set.key + ".png";
+    }
+
+    public enum Set {
```

Comment:
Not sure about this.

1) I'm not a fan of the Set classname, it's confusing with the collections Set.
2) It's now directly tied to robohash, which is maybe not ideal. We had to change the avatar service before, I prefer not to introduce something which may break in the future.

Also, it's not really clear to me: why should we need this functionality?